compiler/src/dotty/tools/dotc/typer/Applications.scala

```diff
@@ -2203,7 +2203,7 @@ trait Applications extends Compatibility {
    *  This reset is needed because otherwise the original results might
    *  have added constraints to type parameters which are no longer
    *  implied after harmonization. No essential constraints are lost by this because
-   *  the result of harmomization will be compared again with the expected type.
+   *  the result of harmonization will be compared again with the expected type.
    *  Test cases where this matters are in pos/harmomize.scala.
```

Comment:
Btw: there is no file `pos/harmomize.scala`, only a `/tests/neg/harmonize.scala`.